From 293e6841fd87cc5988ce83fd691c3646f9f7b19c Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 27 Jul 2021 13:06:13 +0300
Subject: [PATCH] [SW-50527] gaudi: security: update dbg comment

We have got new information from VLSI regarding the problem we observe
during protecting the DBG EML TPC page.
Update the comment with new data.
No functional changes.

Change-Id: I28f39f7f85595dfbe5898c9f3d10f082f1bf2136
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index e485cb7..707cabc 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1022,10 +1022,10 @@
 		regmap_writel(0, prot_base + 0xEF8);
 		/*
 		 * The Protection Bits register protecting the privilege bits
-		 * must be accessed the last due to apparent bug in h/w, which
-		 * according to observation changes (strips?) the privilege
-		 * level from or access transaction, so further accesses result
-		 * in exception.
+		 * must be accessed the last due to DBG protocol not having the
+		 * protection bits, therefore the privilege level here does not
+		 * play any role on access. So further accesses (no matter the
+		 * initiator) will result in exception.
 		 */
 		regmap_writel(0, prot_base + 0xEF4);
 
